### PR TITLE
State interval connection

### PR DESCRIPTION
**Pull request by nathanstilwell**
Another part of #24

### CODE REVIEW

File: lib/utils/parse-options.js [hunk 1]
```diff
@@ -31,10 +35,19 @@ function parseOptions (options) {
     options.cache.timeout = DEFAULTS.cache.timeout
   }
 
+  if (options.interval && typeof options.interval === 'number') {
```
Comment (gr2m) [inline lib/utils/parse-options.js]: I think we could simply do just `if (typeof options.interval === 'number')` here, right?

Comment (nathanstilwell) [inline lib/utils/parse-options.js]: Good point. :smile: Old habits, amirite?

**Comment by gr2m** [comment]
besides https://github.com/hoodiehq/hoodie-client-connection-status/pull/44#discussion_r54200613, LGTM :+1:

**Comment by gr2m** [comment]
[this one’s for you](https://www.youtube.com/watch?v=dgvQnXxT3f4) :dancers:

**Pull request merged**